kitsune/landings/jinja2/landings/home.html

```diff
@@ -1,11 +1,6 @@
-{% extends "base.html" %}
-{% from 'products/includes/product_macros.html' import product_cards with context %}
-{% set styles = ('home',) %}
-{% set canonical_url = canonicalize(viewname='home') %}
-
-{% block hidden_search_masthead %}{% endblock %}
-
-{% block contentwrap %}
+{% extends "base.html" %} {% from 'products/includes/product_macros.html' import product_cards with
```

Comment:
This multiline doesn't look very nice

Reply:
That's formatter's default behavior. I will tune it a bit